test/benchmarks/material/radio/radio.perf-spec.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {$, browser} from 'protractor';
+import {runBenchmark} from '@angular/dev-infra-private/benchmark/driver-utilities';
+
+describe('radio button performance benchmarks', () => {
+  beforeAll(() => {
+    browser.rootEl = '#root';
+  });
+
+  it('renders two radio buttons', async() => {
+    await runBenchmark({
+      id: 'render-two-radio-buttons',
+      url: '',
+      ignoreBrowserSynchronization: true,
```

Comment:
Out of curiosity: Should we disable browser synchronization by default? We set it quite often and it feels unnecessary to repeat that everywhere? Surely can be a follow-up 😄

Reply:
Sure, should be a simple change 👍